tests/monster_attack_test.cpp

```diff
@@ -182,13 +182,13 @@ TEST_CASE( "monster_throwing_sanity_test", "[throwing],[balance]" )
     const std::string monster_type = "mon_feral_human_pipe";
     monster &test_monster = spawn_test_monster( monster_type, attacker_location );
     test_monster.set_goal( target_location );
-    const mtype_special_attack &attack = test_monster.type->special_attacks.at( "gun" );
+    const mtype_special_attack &attack = test_monster.type->special_attacks.at( "throw" );
     reset_caches( attacker_location.z, target_location.z );
     statistics<int> damage_dealt;
     do {
         you.set_all_parts_hp_to_max();
```

Comment:
The monster's running out of moves to attack with.
```suggestion
        test_monster.moves = test_monster.type->speed;
        you.set_all_parts_hp_to_max();
```